rust-arroyo/src/utils/timing.rs

```diff
@@ -7,6 +7,7 @@ pub struct Deadline {
     duration: Duration,
 }
 
+#[inline]
```

Comment:
For dumb cases like this I would go as far as forcing `#[inline(always)]`. I would not default to this but for direct call through functions I think it's a better policy.

Reply:
I updated the qualifier. Thanks for the quick review.